_includes/body/analytics.html

```diff
@@ -1,43 +1,43 @@
 {% if site.google_analytics %}
-  <script>!function(w, d) {
-    w.ga=w.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+<script>!function (w, d) {
+        window.dataLayer = w.dataLayer || [];
+        function gtag() {dataLayer.push(arguments);}
+        gtag('js', new Date());
 
-    /*{% if site.hydejack.cookies_banner %}*/
-      if (navigator.CookiesOK) {
-        ga('create', '{{ site.google_analytics }}', 'auto');
-      } else if (d.cookie.indexOf("hy--cookies-ok=true") > -1) {
-        ga('create', '{{ site.google_analytics }}', {
-          'storage': 'none',
-          'clientId': localStorage ? localStorage.getItem('ga--client-id') : undefined
-        });
-      } else {
-        ga('create', '{{ site.google_analytics }}', {
-          'storage': 'none'
-        });
-        ga('set', 'forceSSL', true);
-        ga('set', 'anonymizeIp', true);
-      }
-    /*{% else %}*/
-      ga('create', '{{ site.google_analytics }}', 'auto');
-    /*{% endif %}*/
+        function gtagConsentDefaultOff() {
+            gtag('consent', 'default', {
+                'ad_storage': 'denied',
+                'analytics_storage': 'denied'
+            });
+        };
+
+        function gtagConsentUpdateOn() {
+            gtag('consent', 'update', {
+                'ad_storage': 'granted',
+                'analytics_storage': 'granted'
+            });
+        };
 
-    var pushStateEl = d.getElementById('_pushState');
-    var timeoutId;
-    pushStateEl.addEventListener('hy-push-state-load', function() {
-      w.clearTimeout(timeoutId);
-      timeoutId = w.setTimeout(function() {
-        ga('set', 'page', w.location.pathname);
-        ga('send', 'pageview');
-      }, 500);
-    });
```

Comment:
GA4 watches the pushState API on it's own so I believe this is no longer needed.  However I believe the feature has to be enabled as part of "Enhanced Metrics" in order to work.  At least on my account it appeared to be enabled by default. 
It may be worth documenting this.
![Screen Shot 2023-09-11 at 1 35 53 PM](https://github.com/hydecorp/hydejack/assets/1916144/5bbfcbce-f849-4b59-a85e-3efb8494499a)